### PR TITLE
Add Japanese translation

### PR DESCRIPTION
**Pull request by xdcyw**
add Japanese translation.

### CODE REVIEW

**Comment by helloworld1** [comment]
Thanks for your contribution. One minor comment is those masked as translatable="false" do not need to be in the translation file.

**Comment by xdcyw** [comment]
OK.  Made some changes, including removal of translatable="false".

File: app/src/main/res/values-ja/string.xml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    <string name="digits">Digits</string>
+    <string name="save">保存</string>
+    <string name="code_copied">コードをクリップボードにコピーしました。</string>
+    <string name="about_freeotp">FreeOTP+について</string>
+    <string name="about_version">FreeOTP+のバージョン %1$s (%2$d)</string>
+    <string name="about_copyright">© 2013 - Red Hat, Inc., et al.\n© 2016 - Liberty</string>
+    <string name="about_license">FreeOTP+ is licensed under %1$s.</string>
+    <string name="about_website">詳細はこちら %s.</string>
+    <string name="about_feedback">We welcome your feedback:&lt;br/&gt;• %1$s&lt;br/&gt;• %2$s</string>
+<<<<<<< HEAD
```
Comment (6543) [inline app/src/main/res/values-ja/string.xml:28]: this merge conflict was not solved and now break locale file at all

**Comment by 6543** [comment]
should I create a pull from crowdin ( did uploade your translations there ...)

**Comment by helloworld1** [comment]
I fixed the issue in the master to unblock the compilation.

**Comment by 6543** [comment]
@xdcyw @helloworld1 -> #161

**Pull request merged**